job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/triggers/TriggerContextSpec.groovy

```diff
@@ -470,10 +470,74 @@ class TriggerContextSpec extends Specification {
                 children().size() == 1
                 with(children()[0]) {
                     name() == 'org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus'
-                    children().size() == 4
+                    children().size() == 5
                     commitStatusContext[0].value() == 'Deploy to staging site'
                     triggeredStatus[0].value() == 'deploy triggered'
                     startedStatus[0].value() == 'deploy started'
+                    statusUrl[0].value() == ''
+                    with(completedStatus[0]) {
+                        children().size() == 2
+                        with(children()[0]) {
+                            name() == 'org.jenkinsci.plugins.ghprb.extensions.comments.GhprbBuildResultMessage'
+                            children().size() == 2
+                            result[0].value() == 'SUCCESS'
+                            message[0].value() == 'All is well'
+                        }
+                        with(children()[1]) {
+                            name() == 'org.jenkinsci.plugins.ghprb.extensions.comments.GhprbBuildResultMessage'
+                            children().size() == 2
+                            result[0].value() == 'FAILURE'
+                            message[0].value() == 'Something has gone wrong'
+                        }
+                    }
+                }
+            }
+        }
+        1 * mockJobManagement.requirePlugin('ghprb')
+        1 * mockJobManagement.requireMinimumPluginVersion('ghprb', '1.26')
+        1 * mockJobManagement.logPluginDeprecationWarning('ghprb', '1.26')
+    }
+    
+    def 'call pull request trigger with commit status extension with custom URL'() {
```

Comment:
don't add an extra test case for the new option, just add the option to the "call pull request trigger with commit status extension" test above.